Libraries/src/Amazon.Lambda.LexV2Events/LexV2Response.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Amazon.Lambda.LexV2Events
+{
+    /// <summary>
+    /// This class represents response from the Lambda function.
+    /// https://docs.aws.amazon.com/lexv2/latest/dg/lambda.html#lambda-response-format
```

Comment:
another JSON to POCO, it should be tested end to end.

Reply:
Tested by returning `LexV2Response` having `DialogAction` (refer https://github.com/aws-samples/amazon-lex-v2-dialogaction/blob/main/lex-validate-lambda/searchHotel/ResponseHandler.js). Updated Readme example since during testing, it appears that `DialogAction` is mandatory in response.